libs/SalesforceSDKCore/SalesforceSDKCore/Classes/RestAPI/SFNetwork.h

```diff
@@ -30,32 +30,73 @@
 #import <Foundation/Foundation.h>
 #import "SalesforceSDKConstants.h"
 
+SFSDK_DEPRECATED(7.3, 8.0, "Will be removed")
 NS_SWIFT_NAME(NetworkManaging)
 @protocol SFNetworkSessionManaging
-- (nonnull NSURLSession *)ephemeralSession:(nonnull NSURLSessionConfiguration *)sessionConfig;
-- (nonnull NSURLSession *)backgroundSession:(nonnull NSURLSessionConfiguration *)sessionConfig;
+- (nonnull NSURLSession *)ephemeralSession:(nonnull NSURLSessionConfiguration *)sessionConfig SFSDK_DEPRECATED(7.3, 8.0, "Use sharedInstanceWithIdentifier:sessionConfiguration instead");
+- (nonnull NSURLSession *)backgroundSession:(nonnull NSURLSessionConfiguration *)sessionConfig SFSDK_DEPRECATED(7.3, 8.0, "Use sharedInstanceWithIdentifier:sessionConfiguration instead");
 @end
 
+extern NSString * __nonnull const kSFNetworkEphemeralInstanceIdentifier NS_SWIFT_NAME(SFNetworkEphemeralInstanceIdentifier);
```

Comment:
Minor comment: `Swift` name should probably be `NetworkEphemeralInstanceIdentifier` (no prefixes in `Swift`, right @trooper2013?).